README.md

```diff
@@ -145,11 +145,11 @@ To test, you will use a Node.js script (`bin/msg.mjs`). Follow these steps:
 
 1. Create a user at the Cognito user pool while defining ther user's password:
 
-   `node bin/msg.mjs create-user -p <password>`
+   `node bin/msg.mjs create-user -u <username> -p <at-least-8-chars-password-with-1-number-and-1-spec-char>`
```

Comment:
About the "-p", let's add the information about the password limitations in the instruction instead at the command line example.

Reply:
Sure thing! I thought maybe we could set it like this, but it looks better as an instruction